glidex.forms/IGlideHandler.cs

```diff
@@ -19,5 +19,6 @@ public interface IGlideHandler
 		/// <param name="token">The CancellationToken if you need it</param>
 		/// <returns>True if the image was handled. Return false if you need the image to be cleared for you.</returns>
 		bool Build (ImageView imageView, ImageSource source, RequestBuilder builder, CancellationToken token);
+		void Build (ImageSource source, RequestBuilder builder, CancellationToken token);
```

Comment:
This should be a C# 8 default interface method, so that existing `IGlideHandler` implementations won't get a compilation error.

This should also use `ref RequestBuilder builder` so that `IGlideHandler`'s can set the `builder` to `null` if desired.